packages/aws-cdk-lib/aws-lambda-nodejs/test/bundling.test.ts

```diff
@@ -11,8 +11,8 @@ import { PackageInstallation } from '../lib/package-installation';
 import { Charset, LogLevel, OutputFormat, SourceMapMode } from '../lib/types';
 import * as util from '../lib/util';
 
-const STANDARD_RUNTIME = Runtime.NODEJS_18_X;
-const STANDARD_TARGET = 'node18';
+const STANDARD_RUNTIME = Runtime.NODEJS_20_X;
+const STANDARD_TARGET = 'node20';
```

Comment:
These test changes are fixed in latest commit to update only expected target value for nodejs latest runtime.